treescript/test/test_versionmanip.py

```diff
@@ -77,6 +77,8 @@ def test_replace_ver_in_file_invalid_old_ver(repo_context, new_version):
     bump_info = {'files': relative_files, 'next_version': new_version}
     mocked_bump_info = mocker.patch.object(vmanip, 'get_version_bump_info')
     mocked_bump_info.return_value = bump_info
+    mocked_dontbuild = mocker.patch.object(vmanip, 'get_dontbuild')
+    mocked_dontbuild.return_value = True
```

Comment:
I was more thinking of a test with and without dontbuild being true, which validates that DONTBUILD_MSG is in the message arg...

So something like this test with something like:
```
command = called_args[0][0]
# Find commit message
commit_msg = command[command.index('-m') + 1]
assert DONTBUILD_MSG in commit_msg
```

and similar for the false state of this (so we test that its not included)

Reply:
We should similarly perform this dontbuild test with the tagging tests